utils/libafl_benches/benches/rand_speeds.rs

```diff
@@ -1,25 +1,16 @@
 //! Compare the speed of rand implementations
 
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
-use libafl_bolts::rands::{
-    Lehmer64Rand, Rand, RomuDuoJrRand, RomuTrioRand, Sfc64Rand, XorShift64Rand,
-    Xoshiro256PlusPlusRand,
-};
+use libafl_bolts::rands::{Rand, RomuDuoJrRand, Sfc64Rand, Xoshiro256PlusPlusRand};
 
 fn criterion_benchmark(c: &mut Criterion) {
     let mut sfc64 = Sfc64Rand::with_seed(1);
-    let mut xorshift = XorShift64Rand::with_seed(1);
     let mut xoshiro = Xoshiro256PlusPlusRand::with_seed(1);
     let mut romu = RomuDuoJrRand::with_seed(1);
-    let mut lehmer = Lehmer64Rand::with_seed(1);
-    let mut romu_trio = RomuTrioRand::with_seed(1);
 
     c.bench_function("sfc64", |b| b.iter(|| black_box(sfc64.next())));
-    c.bench_function("xorshift", |b| b.iter(|| black_box(xorshift.next())));
     c.bench_function("xoshiro", |b| b.iter(|| black_box(xoshiro.next())));
     c.bench_function("romu", |b| b.iter(|| black_box(romu.next())));
-    c.bench_function("romu_trio", |b| b.iter(|| black_box(romu_trio.next())));
```

Comment:
Why would you remove these guys?

Reply:
RomuTrio I would probably keep, just because it's romu but more precise.
Xorshift is the default for many use cases, so it cold make sense to keep it for comparison reasons (part of why LibAFL exists)
Lehmer I don't even know why we have it :D

Reply:
Ok, I can leave RomuTrio and Xorshift (with a note that it is mainly only included for comparison purposes).